app.rb

```diff
@@ -1,16 +1,25 @@
+# frozen_string_literal: true
```

Comment:
Which linter introduced this? afaik [govuk-lint](https://github.com/alphagov/govuk-lint) doesn't have this rule, so I'd be inclined to not start introducing it to follow convention.